editor/editor_node.cpp

```diff
@@ -541,7 +542,8 @@ void EditorNode::_update_update_spinner() {
 	update_popup->set_item_checked(update_popup->get_item_index(SETTINGS_UPDATE_CONTINUOUSLY), update_continuously);
 	update_popup->set_item_checked(update_popup->get_item_index(SETTINGS_UPDATE_WHEN_CHANGED), !update_continuously);
 
-	OS::get_singleton()->set_low_processor_usage_mode(!update_continuously);
+	/* OS::get_singleton()->set_low_processor_usage_mode(!update_continuously); */
+	OS::get_singleton()->set_low_processor_usage_mode(false);
```

Comment:
Can you explain what the changes in this file in line 392 and line 546 are for?

Reply:
and 392 is don't quit on back, it looks like it different from desktop `set_auto_accept_quit`, so android needs to disable that also

Reply:
Actually testing `OS::get_singleton()->set_low_processor_usage_mode(!update_continuously);` on my oneplus 5 seems to work OK, @fahrstuhl mind explaining why you did that, is your device older?

Reply:
Yeah, I did that because it did not work on my device, [as I mentioned in the issue](https://github.com/godotengine/godot/issues/267#issuecomment-571190195). I can't test the current version because my phone does not support Vulkan.